dataset_config/1000_genomes/ui.json

```diff
@@ -69,5 +69,5 @@
         "samples.verily-public-data.human_genome_variants.1000_genomes_sample_info.Main_project_LC_platform"
     }
   ],
-  "enable_field_search": false
+  "enable_field_search": true
```

Comment:
Hmm, good point that this needs to be enabled. In retrospect I didn't put the right order in the spreadsheet.

Let's do these before enabling by default:

- A PR to fix chip text. (First 10 characters looks weird.)
- A PR to fix SQL export, since current SQL is incorrect

Then we can proceed:
- A PR to remove enable_field_search flag. You want to undo relevant parts of #142.
- Then I'll review this PR, and this PR can go in.

Reply:
Actually, "PR to remove enable_field_search flag" can come after this PR if you want. I just want the first 2 PRs to be in